e2e/setup-local-fdb-cluster.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# Set up local FDB non-HA cluster on a 4-node kind k8s cluster
+# This only works on x86 machines as FDB doesn't provide arm64 Linux binaries yet
+# Assumptions: (1) There is at most one kind cluster in the env; (2) kubectl is pointing to the kind cluster
+
+if [ "$#" -le 1 ]; then
+read -p "create kind cluster? (enter yes or no): " createKindCluster
+read -p "enter k8s node version? (e.g., v1.23.0): " version
+else
+    createKindCluster=$1
+    version=${2}
+fi
+
+cluster=${cluster:-"local-cluster"}
+
+if [ "${createKindCluster}" = "yes" ]; then
+    echo "===Start creating k8s cluster on kind"
+    cat <<EOF | kind create cluster --name ${cluster} --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  image: kindest/node:${version}
+- role: worker
+  image: kindest/node:${version}
+- role: worker
+  image: kindest/node:${version}
+- role: worker
+  image: kindest/node:${version}
+EOF
+
+else
+    echo "===Skip creating k8s cluster on kind"
+    # TODO: make sure kind is using the local-cluster context
+    kind get clusters
+fi
+
+echo "===Start building operator"
+echo "---We should be at the repo's root directory: "
+pwd
+
+./config/test-certs/generate_secrets.bash
+make rebuild-operator
+
+echo "===Load operator image to kind cluster"
+kind load docker-image "fdb-kubernetes-operator:latest" --name ${cluster}
+
+echo "===Creating a FDB cluster with the FDB operator"
+kubectl apply -k ./config/tests/base
+
+echo "===Done==="
+
+# TODO: Make sure kubectl context is pointing to the kind cluster
+kubectl get fdb
+echo "Waiting for creating FDB Pods..."
+sleep 10;
+kubectl wait --for=condition=ready pod -l foundationdb.org/fdb-cluster-name=test-cluster
+kubectl get fdb
```

Comment:
I'm not sure how useful this part here is and if that should be all in this script. Based on the CI output it doesn't look like anything is happening:

```bash
===Done===
[2712](https://github.com/FoundationDB/fdb-kubernetes-operator/actions/runs/4378195162/jobs/7662624682#step:7:2713)
NAME           GENERATION   RECONCILED   AVAILABLE   FULLREPLICATION   VERSION   AGE
[2713](https://github.com/FoundationDB/fdb-kubernetes-operator/actions/runs/4378195162/jobs/7662624682#step:7:2714)
test-cluster   1                                                                 0s
[2714](https://github.com/FoundationDB/fdb-kubernetes-operator/actions/runs/4378195162/jobs/7662624682#step:7:2715)
Waiting for creating FDB Pods...
[2715](https://github.com/FoundationDB/fdb-kubernetes-operator/actions/runs/4378195162/jobs/7662624682#step:7:2716)
error: no matching resources found
[2716](https://github.com/FoundationDB/fdb-kubernetes-operator/actions/runs/4378195162/jobs/7662624682#step:7:2717)
NAME           GENERATION   RECONCILED   AVAILABLE   FULLREPLICATION   VERSION   AGE
[2717](https://github.com/FoundationDB/fdb-kubernetes-operator/actions/runs/4378195162/jobs/7662624682#step:7:2718)
test-cluster   1                                                                 10s
```